webserver/web-interface/src/pages/home/Home.jsx

```diff
@@ -81,8 +82,24 @@ function Home() {
     };
   }, []);
 
+  function handleRemove() {
+    const targetBox = document.getElementById('targetBox');
+    const currentValue = parseInt(targetBox.value);
+    const newValue = currentValue - 1;
+    targetBox.value = newValue.toString();
+  }
+
+  function handleAdd() {
+    const targetBox = document.getElementById('targetBox');
+    const currentValue = parseInt(targetBox.value);
+    const newValue = currentValue + 1;
+    targetBox.value = newValue.toString();
+
+  }
```

Comment:
this is a vanilla JS approach to things. We shouldn't need to use `getElementById` unless we're escape hatching from React.

Instead the target value can be state defaulted to `93` and the value can be interpolated and added to below.

This then just becomes 
```jsx
const [targetValue, setTargetValue] = useState(93);

function handleAdd() {
  setTargetValue(oldValue => oldValue + 1);
}

function handleRemove() {
  setTargetValue(oldValue => oldValue - 1);
}
```

Reply:
This way it looks nicer and better indeed, the current value handling ain't what's needed, i was just mocking smth there until  the actual target var is dropped in there and handled dynamically between esp and bp